tests/vit_mae/test_modeling_vit_mae.py

```diff
@@ -319,6 +319,8 @@ def check_hidden_states_output(inputs_dict, config, model_class):
 
             check_hidden_states_output(inputs_dict, config, model_class)
 
+    # overwrite from common since ViTMAEForPretraining has random masking, we need to fix the noise
+    # to generate masks during test
```

Comment:
just a nit

Reply:
Kept it intentionally to explain the purpose of the override.